tests/nosetests/pyanaconda_tests/product_test.py

```diff
@@ -29,12 +35,106 @@
 
 PRODUCT_DIR = os.path.join(os.environ.get("ANACONDA_DATA"), "product.d")
 
+SERVER_PARTITIONING = [
```

Comment:
The main reason why these constants are moved to the tests is that it is the easiest way to validate this configuration option. This is a big change with many small details that could be easily overlooked, so it is great that we can confirm that the new option will create the same partitioning as before. We might want to rework this part in the future, but there will be always some hard coded values, because you can not really test something without them. I don't see how tests can enforce something.